docs/supported-mutators.md

```diff
@@ -121,32 +123,32 @@ Stryker.NET _specific mutator_
 
 ## Equality Operator
 
-| Original  | Mutated     |
-| --------- | ----------- |
-| `a < b`   | `a <= b`    |
-| `a < b`   | `a >= b`    |
-| `a <= b`  | `a < b`     |
-| `a <= b`  | `a > b`     |
-| `a > b`   | `a >= b`    |
-| `a > b`   | `a <= b`    |
-| `a >= b`  | `a > b`     |
-| `a >= b`  | `a < b`     |
-| `a == b`  | `a != b`    |
-| `a != b`  | `a == b`    |
-| `a === b` | `a !== b` ¹ |
-| `a !== b` | `a === b` ¹ |
+| Mutant operator                  | Original  | Mutated     |
+| -------------------------------- | --------- | ----------- |
+| LessThanOperatorBoundary         | `a < b`   | `a <= b`    |
+| LessThanOperatorNegation         | `a < b`   | `a >= b`    |
+| LessThanEqualOperatorBoundary    | `a <= b`  | `a < b`     |
+| LessThanEqualOperatorNegation    | `a <= b`  | `a > b`     |
+| GreaterThanOperatorBoundary      | `a > b`   | `a >= b`    |
+| GreaterThanOperatorNegation      | `a > b`   | `a <= b`    |
+| GreaterThanEqualOperatorBoundary | `a >= b`  | `a > b`     |
+| GreaterThanEqualOperatorNegation | `a >= b`  | `a < b`     |
+| EqualityOperatorNegation         | `a == b`  | `a != b`    |
+| InequalityOperatorNegation       | `a != b`  | `a == b`    |
+| StrictEqualityOperatorNegation   | `a === b` | `a !== b` ¹ |
+| StrictInequalityOperatorNegation | `a !== b` | `a === b` ¹ |
 
 - ¹: Only supported on StrykerJS and Stryker4s
 
 [🔝 Back to Top](#supported-mutators)
 
 ## Logical Operator
 
-| Original                      | Mutated                       |
-| ----------------------------- | ----------------------------- |
-| `a && b`                      | <code>a &vert;&vert; b</code> |
-| <code>a &vert;&vert; b</code> | `a && b`                      |
-| `a ?? b`                      | `a && b`                      |
+| Mutant operator                                  | Original                      | Mutated                       |
+| ------------------------------------------------ | ----------------------------- | ----------------------------- |
+| LogicalAndOperatorNegation                       | `a && b`                      | <code>a &vert;&vert; b</code> |
+| LogicalOrOperatorNegation                        | <code>a &vert;&vert; b</code> | `a && b`                      |
+| NullishCoalescingOperatorToLogicalAndReplacement | `a ?? b`                      | `a && b`                      |
```

Comment:
"Nullish" is not a concept in the .NET world, and I wouldn't know what to expect from this. Although I don't have a better idea, maybe only something even more verbose like "NullNullishCoalescingOperatorToLogicalAndReplacement".

(just passing by here - actually I think all this PR is a good idea)

Reply:
I don't think Nullish makes sense anyway. The end result is that the value is treated in such a way that it is the same as Null, so NullCoalescing is good enough imo.